eng/pipelines/runtimelab.yml

```diff
@@ -57,22 +57,27 @@ stages:
   #
   - template: /eng/pipelines/common/checkout-job.yml
 
-#
-# Build with Release config and Release runtimeConfiguration
-#
-- template: /eng/pipelines/common/platform-matrix.yml
-  parameters:
-    jobTemplate: /eng/pipelines/common/global-build-job.yml
-    buildConfig: Release
-    platforms:
-    - Linux_x64
-    - FreeBSD_x64
-    jobParameters:
-      testGroup: innerloop
-      buildArgs: -s clr+libs+libs.tests+installer -c $(_BuildConfig) /p:ArchiveTests=true
-      extraStepsTemplate: /eng/pipelines/runtimelab/runtimelab-post-build-steps.yml
-      extraStepsParameters:
-        uploadTests: true
+  #
+  # Build with Release config and Release runtimeConfiguration
+  #
+  - template: /eng/pipelines/common/platform-matrix.yml
+    parameters:
+      jobTemplate: /eng/pipelines/common/global-build-job.yml
+      buildConfig: Release
+      platforms:
+      - Linux_x64
+      - FreeBSD_x64
+      jobParameters:
+        isOfficialBuild: ${{ variables.isOfficialBuild }}
+        testGroup: innerloop
+        extraStepsTemplate: /eng/pipelines/runtimelab/runtimelab-post-build-steps.yml
+        extraStepsParameters:
+          uploadLibrariesTests: ${{ eq(variables.isOfficialBuild, false) }}
+          uploadUnsignedArtifacts: false
+        ${{ if eq(variables.isOfficialBuild, false) }}:
+          buildArgs: -s clr+libs+libs.tests+installer -c $(_BuildConfig) /p:ArchiveTests=true
```

Comment:
It seems like you don't need archiveTests=true here since we're not running tests.

Reply:
for this one I'm less sure. In ideal case we would be running tests. I'm sort of hoping that if we build and publish tests, it would be easier to download and run everything outside of Helix. Do you have sense how much space does it take?

Reply:
Hmm, I have no idea, but we should be able to look at the artifact that we publish with the libraries tests in it.